services/forms-web-app/package.json

```diff
@@ -27,7 +27,8 @@
     "http-errors": "~1.6.3",
     "morgan": "~1.9.1",
     "node-sass": "^4.14.1",
-    "nunjucks": "^3.2.2"
+    "nunjucks": "^3.2.2",
+    "pino": "^6.7.0"
```

Comment:
If we're using `pino` (good choice), do we need `debug` in? It'll probably make life easier having just one logging mechanism in. It might be worth getting rid of morgan and using pino for middleware too with the [pino-express-logger](https://github.com/pinojs/express-pino-logger) middleware

Reply:
Added pino request logging using pino-express, remove debug.